### ISSUE TITLE
LR Config cannot be set by AutoML

### ISSUE THREAD

**Issue by AmoghDabholkar** [NVIDIA/tao_front_end_services#3]
**Describe the bug**
LR config cannot be set by using AutoML, as these parameter are missing from the automl classification training csv.

**Steps/Code to reproduce bug**
Run AutoML Jupyter notebook and run the cell including "additional automl parameters"

**Expected behavior**
[train.csv](https://github.com/NVIDIA/tao_front_end_services/blob/main/api/specs_utils/specs/classification_tf2/classification_tf2%20-%20train.csv) should have the option to set LR config, in which case I am not sure if setting train.optim_config.lr in AutoML parameters matters because train.lr_config.learning_rate sets the learning rate in general right? So does varying train.optim_config.lr make any difference?

Environment overview (please complete the following information)
TAO v5.0.0

 - Environment location: [Bare-metal, Cloud(specify cloud provider - AWS, Azure, GCP, Collab)]
Bare-metal

Raised an issue on the NVIDIA discussion forum as well 
https://forums.developer.nvidia.com/t/tao-5-0-training-spec-discrepancy/261123/5

**Comment by Arun-George-Zachariah** [comment]
Marking this issue as closed as it's been addressed in the [Forum](https://forums.developer.nvidia.com/t/tao-5-0-training-spec-discrepancy/261123).
